sdk/cosmos/azure-cosmos-spark_3-0_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConstants.scala

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.spark
+
+// cosmos db related constants
+private object CosmosConstants {
+  val CosmosIdFieldName = "id"
+
+  object StatusCodes {
+    val ConflictStatusCode = 409
```

Comment:
Doesn't Scala provide the HTTP constants already?

Reply:
These are Cosmos DB specific status code, I would rather not to depend on an unrelated third party scala http package for the cosmos db status codes.

Reply:
Got it, I was just wondering, in .NET, the framework provides the list of enums for the common HTTP codes. Thought that since these are common HTTP codes, Scala might provide the enum values.